pkg/actuators/machineset/controller_test.go

```diff
@@ -94,21 +92,12 @@ var _ = Describe("MachineSetReconciler", func() {
 			return make(map[string]string)
 		}, timeout).Should(Equal(rtc.expectedAnnotations))
 
-		// Check which event types were sent
-		Eventually(fakeRecorder.Events, timeout).Should(HaveLen(len(rtc.expectedEvents)))
```

Comment:
Might still be worth sending an event, how hard would it be to plumb an event recorder through to the logic here so we can send an event?